### ISSUE TITLE
Allow using the `--quiet` flag in the newsboat config file

### ISSUE THREAD

**Issue by bbhtt** [newsboat/newsboat#1930]
**Newsboat version**:
```
$ newsboat -v

Newsboat 2.26.0
System: Linux 5.15.13-arch1-1 (x86_64)
Compiler: g++ 11.1.0
ncurses: ncurses 6.3.20211021 (compiled with 6.3)
libcurl: libcurl/7.81.0 OpenSSL/1.1.1m zlib/1.2.11 brotli/1.0.9 zstd/1.5.1 libidn2/2.3.2 libpsl/0.21.1 (+libidn2/2.3.0) libssh2/1.10.0 nghttp2/1.46.0 (compiled with 7.80.0)
SQLite: 3.37.2 (compiled with 3.37.0)
libxml2: compiled with 2.9.12
```

Currently to my understanding, Newsboat doesn't have a way to use the `--quiet` [flag](https://newsboat.org/releases/2.26/docs/newsboat.html#_first_steps) from the config file. It'd be useful to have it in the config file when one wants to silence output (instead of carrying aliases or specifying it on every run.)

**Steps to reproduce the issue**: N/A

**Other info you think is relevant**: N/A

**Comment by bogdasar1985** [comment]
I can try to make this issue, but before I should finish this PR(https://github.com/newsboat/newsboat/pull/1925)

**Comment by Minoru** [comment]
@bbhtt This got implemented in #1941, but that PR showed a chicken-and-egg problem: on startup, Newsboat wants to say that it started loading the config, but it can't do that until it has finished loading the config and checked the `quiet` setting. This can be hidden by reordering the code as is done in #1941, but I think it makes the already-complicated startup code even more treacherous.

Please use a shell alias instead.
